src/woo_publications/api/models.py

```diff
@@ -0,0 +1,69 @@
+import binascii
+import os
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from django_jsonform.models.fields import JSONField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from .constants import PermissionOptions
+
+PERMISSION_SCHEMA = {
+    "type": "list",
+    "items": {
+        "type": "string",
+        "choices": [
+            {"title": label, "value": value}
+            for value, label in PermissionOptions.choices
+        ],
+        "widget": "multiselect",
+    },
+}
+
+
+class TokenAuth(models.Model):
```

Comment:
I thought we had discussed that the model name should be `Application`? You created an app `application` then and I told you to put the model in `api.models`, but it should still be named `Application` because that's what it represents.

This keeps terminology close to the OAUTH application concept and should make the admin interface easier to use.

For the admin index menu, I think it makes sense to create a group "API access" containing this model.